README.md

```diff
@@ -4,6 +4,8 @@ A Visual Studio Code test extension that allows users to run their [Kani Rust Ve
 
```

Comment:
Not included in the change, but I think we should change the main description to:

```
A [Visual Studio Code](https://code.visualstudio.com/) extension that allows users to run their [Kani Rust Verifier](https://github.com/model-checking/kani) harnesses and [...] in vscode.
```
It's unclear to me what other features (i.e., what should go in `[...]`) we should highlight. Viewing traces doesn't seems like the most relevant feature to me.